src/Modules/OrchardCore.Commerce/Views/OrderPart.cshtml

```diff
@@ -45,6 +47,18 @@
                 </div>
             </div>
         </div>
+        @if (string.Equals(Model.OrderPart.Status.Text, OrderStatuses.Pending, StringComparison.OrdinalIgnoreCase))
+        {
+            <div class="my-2 d-flex justify-content-center">
+                <div class="pay-button-container">
+                    <a href="@Href($"~/PaymentRequest/{Model.OrderPart.ContentItem.ContentItemId}")" class="pay-button btn btn-success justify-content-center">
```

Comment:
Please use `@Orchard.Action<PaymentController>(controller => controller.PaymentRequest(Model.OrderPart.ContentItem.ContentItemId))` instead of hard coding the route.

Reply:
This returns `/OrchardCore.Commerce/Payment/PaymentRequest`, which is not quite what is necessary (`/checkout/paymentrequest/4kx5ap5ds7qvs3swwtn15m13md`), and it looks like the path cannot be customized to such an extent using this method. Are there any other viable alternatives?

Reply:
Very strange. Use the `asp-*` argument tag helpers then, like
```
                    <a asp-area="OrchardCore.Commerce"
                       asp-controller="@typeof(PaymentController).ControllerName()"
                       asp-action="@nameof(PaymentController.PaymentRequest)"
                       asp-route-orderId="@Model.OrderPart.ContentItem.ContentItemId"
```